Lib/test/test_tarfile.py

```diff
@@ -3056,6 +3057,94 @@ def test_keyword_only(self, mock_geteuid):
                               tarfl.extract, filename_1, TEMPDIR, False, True)
 
 
+_can_chmod_set_sticky = None
+
+
+def can_chmod_set_sticky():
```

Comment:
You should move it to support.os_helper.

I suggest to rename to can_chmod_set_sticky_bit().